package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-responsive-image",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Yes device.js is already updated but the 1.0.0 does not have the update so pushing a 1.0.1 would be great for anyone using the project with the most recent versions of react-native.